Added i18n mechanism and locale file of Japanese.

shinobukawano opened the pull request:
You can change the value of "title", "text" or "name" property by chrome i18n message catalog.

ex)
title:
'About App Inspector for Sencha'
to
'App Inspector for Sencha について'

It allows the message key which contains space value. If doesn't exist a message in the catalog, use it just as it is.

![i18n](https://f.cloud.github.com/assets/321647/2447730/2d30f568-ae9c-11e3-9993-1e648831797f.png)

shinobukawano committed:
Added i18n mechanism and locale file of Japanese.

You can change the value of "title", "text" or "name" property by chrome i18n message catalog.

ex)
title:
'About App Inspector for Sencha'
to
'App Inspector for Sencha について'

It allows the message key which contains space value. If doesn't exist a message in the catalog, use it just as it is.

arthurakay commented:
I'll need @stoe to comment on the i18n bits, since he is our resident multi-linguist :-)

I think it looks good conceptually.

@kawanoshinobu Do add yourself to the list of contributors in CONTRIBUTORS.md and app/AppInspector/about.html - we really appreciate the PR!

shinobukawano committed:
Add my name to the contributors list. I'm honored :)

shinobukawano commented:
Hi @arthurakay,

Thank you for the comment! I'm happy to add my name to the list. Thank you very much :)
I wish I contribute to the project continuously. Please keep in touch!

shinobukawano committed:
Added some messages that were overlooked.

- Utility to find components which may be overnested inside the application.
- Utility to find nested box layouts inside the application which may contribute to performance issues.

arthurakay commented:
@stoe Anything else need to be done for the locales?

stoe commented:
@arthurakay not at the moment.
I think we might need to port it to be a mixin instead of an override at some point in time, but that doesn't yet work together with Architect.
